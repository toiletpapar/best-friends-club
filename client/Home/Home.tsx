import * as React from 'react'

import { Nav } from '../Nav/index'

const Home = () => {
  return (
    <Nav />
  )
}

export {
  Home
}
