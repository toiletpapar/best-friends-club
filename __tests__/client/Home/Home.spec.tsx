import * as React from 'react'
import * as renderer from 'react-test-renderer'
import { Home } from '../../../client/Home/index'

describe('#Home', (): void => {
  it('should render correctly', (): void => {
    const tree = renderer
      .create(<Home />)
      .toJSON()

    expect(tree).toMatchSnapshot()
  })
})