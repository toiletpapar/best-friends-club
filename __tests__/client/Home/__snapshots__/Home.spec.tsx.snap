// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Home should render correctly 1`] = `
<div
  className="sc-dxgOiQ fnNCjc"
>
  <div
    className="sc-jKJlTe jqwgMa sc-kpOJdX jSpUW"
  >
    <img
      className="sc-kGXeez jzbGIe"
      src="test-file-stub"
    />
    <a
      className="sc-dnqmqq jFCsgM"
      href="/"
      onClick={[Function]}
    >
      Lobby
    </a>
  </div>
  <div
    className="sc-ckVGcZ lbASTC"
  >
    <h3
      className="sc-fjdhpX cQEeDA"
    >
      Codenames
    </h3>
    <p>
      Choose a game from the list below or create your own.
    </p>
    <ul
      className="sc-jzJRlG lkdUhq"
    >
      <li
        className="sc-cSHVUG enaHVM"
      >
        Unable to find any games. Why not create one?
      </li>
    </ul>
    <button
      className="sc-iwsKbI izFQUc"
      onClick={[Function]}
    >
      Create Game
    </button>
    <button
      className="sc-iwsKbI jvsbIL"
      onClick={[Function]}
    >
      Join Game
    </button>
  </div>
</div>
`;
